src/connectors/cacheDb/cacheDb.connector.js

```diff
@@ -68,9 +75,204 @@ const cacheRegistration = async registration => {
   }
 };
 
+/**
+ * Update the completed objects for the Registration and Tascomi objects
+ * @param {string} fsa_rn The FSA-RN for the registration to be updated
+ * @param {string} property The specific field to be updated
+ * @param {boolean} value The value for the result
+ */
+const updateStatusInCache = async (fsa_rn, property, value) => {
+  logEmitter.emit("functionCall", "cacheDb.connector", "updateStatusInCache");
+  try {
+    const cachedRegistrations = await establishConnectionToMongo();
+    let status = await getStatus(cachedRegistrations, fsa_rn);
```

Comment:
this can be a const, because status is never re-assigned, you only change its properties